### PR TITLE
Address misspelling in Changelog

### PR DESCRIPTION
**Pull request by Stargator**
Note: I changed "barback" to "bareback" as I thought that was the original intent. If not, I can change it back, obviously.

### CODE REVIEW

**Comment by Stargator** [comment]
I signed it!

File: CHANGELOG.md [hunk 1]
```diff
@@ -733,9 +733,9 @@ Patch release, resolves two issues:
 
 * Pub
 
-  * Avoid using a barback asset server for executables unless they actually use
-    transformers. This makes precompilation substantially faster, produces
-    better error messages when precompilation fails, and allows
+  * Avoid using a bareback asset server for executables unless they actually use
```
Comment (lrhn) [inline CHANGELOG.md]: "Barback" is actually correct. It refers to https://github.com/dart-lang/barback

(Otherwise fine).

Comment (Stargator) [inline CHANGELOG.md]: Fixed

**Review by kevmoo — APPROVED** [review]

**Comment by whesse** [comment]
No pull requests should be landed on the dev branch.  Only merges, made following the process in the wiki: https://github.com/dart-lang/sdk/wiki/Cherry-picks-to-dev-channel

Were these errors not in the Changelog on master?  Won't we overwrite them in our next full push?

**Comment by kevmoo** [comment]
@whesse – total fail on my part. Didn't even see the target branch.

**Comment by Stargator** [comment]
@whesse and @kevmoo I apologize, when I read the [Contributing](https://github.com/dart-lang/sdk/wiki/Contributing#keeping-your-branch-updated-with-originmaster) doc it should have occurred to me to ask about the dev branch since the document only mentioned master. I'll be more mindful in the future.

**Pull request merged**